src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +46,38 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+
+        final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
it isn't constant (you can read about it in slack or https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)